migration/sql-files/111-add-wit-to-trackerquery.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE tracker_queries ADD COLUMN work_item_type_id uuid REFERENCES work_item_types(id);
```

Comment:
I don't see a migration test file for this statement. And Please be specific by adding `ON DELETE CASCADE` for example.

Reply:
Is the column value nullable? If not what will be the value for existing records?

Reply:
@kwk I have added on delete cascade here 68498a6

Reply:
@baijum I have added NOT NULL constraint as well as handled work_item_type_id for existing records here 68498a6

Reply:
@baijum @kwk Based on the discussion,
[1] I remove old tracker_queries records (as we did not support the remote work item feature earlier)
[2] I add `NOT NULL` constraint to work_item_type_id 

97f2247